src/patternfly/components/Page/page.scss

```diff
@@ -272,6 +270,12 @@
     box-shadow: var(--pf-c-page__sidebar--BoxShadow);
   }
 
+  .pf-c-page__sidebar-body {
+    height: 100%;
```

Comment:
is this necessary?

Reply:
I was mimicking what was there before adding the child element.

**Before:**
<img width="401" alt="screen shot 2019-03-05 at 3 10 21 pm" src="https://user-images.githubusercontent.com/20118816/53834396-d8e3f200-3f58-11e9-83a6-cbaee741010b.png">

**After (without height: 100%)**
<img width="356" alt="screen shot 2019-03-05 at 3 09 58 pm" src="https://user-images.githubusercontent.com/20118816/53834472-16e11600-3f59-11e9-9dab-66ec74e96e64.png">

After (with height: 100%)
<img width="416" alt="screen shot 2019-03-05 at 3 10 07 pm" src="https://user-images.githubusercontent.com/20118816/53834575-632c5600-3f59-11e9-89d0-92d8cde4bafc.png">

Reply:
Gotcha, the sidebar needs to be full height because it has styles (like `box-shadow`) that rely on the element being full height. But the body element shouldn't, just like how the nav component inside doesn't. Or do you see a reason that we need it?

Reply:
Ah I don't see a reason that we need it! So good call!